src/main/java/com/exadel/discount/entity/Vendor.java

```diff
@@ -0,0 +1,44 @@
+package com.exadel.discount.entity;
+
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+import javax.persistence.*;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.UUID;
+
+@Getter @Setter
+@EqualsAndHashCode
+@NoArgsConstructor
+@Entity
+@Table(name = "vendors")
+public class Vendor {
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    private UUID id;
+
+    private String name;
+    private String description;
+
+    @OneToMany(cascade = CascadeType.ALL,
+    fetch = FetchType.EAGER,
+    mappedBy = "vendor")
+    private List<VendorLocation> locationVendors = new ArrayList<>();
+
```

Comment:
vendorLocations instead of locationVendors

Reply:
+